optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/report/BenchmarkReport.java

```diff
@@ -220,12 +225,44 @@ public void writeReport() {
         for (ProblemBenchmarkResult problemBenchmarkResult : plannerBenchmarkResult.getUnifiedProblemBenchmarkResultList()) {
             if (problemBenchmarkResult.hasAnySuccess()) {
                 for (ProblemStatistic problemStatistic : problemBenchmarkResult.getProblemStatisticList()) {
+                    for (SingleStatistic singleStatistic : problemStatistic.getSingleStatisticList()) {
+                        try {
+                            singleStatistic.unhibernatePointList();
+                        } catch (IllegalStateException ise) {
+                            if (!plannerBenchmarkResult.getAggregation()) {
+                                throw ise;
```

Comment:
Never throw an exception that is already thrown, as that eats the original stacktrace.
Just chain it:
throw new IllegalStateException("...", ise);